packages/client/utils/isNonEmptyArray.ts

```diff
@@ -0,0 +1 @@
+export default (thing: any) => Array.isArray(thing) && thing.length
```

Comment:
this addresses a bug in `packages/server/graphql/queries/tasks.ts` where it was considering `[]` to be truthy, as js does by default.